### PR TITLE
Main commit getting rid of Quantified dependency - And replacing with Measured 1.6.0

### PR DESCRIPTION
**Pull request by benwah**
The changes are pretty simple - Everything still seems to work.

### CODE REVIEW

File: test/unit/carriers/australia_post_test.rb [hunk 1]
```diff
@@ -11,13 +11,13 @@ def setup
   end
 
   def test_service_domestic_simple_request
-    url = 'https://digitalapi.auspost.com.au/postage/parcel/domestic/service.json?from_postcode=2000&height=2&length=19&to_postcode=3108&weight=0.25&width=14'
+    url = 'https://digitalapi.auspost.com.au/postage/parcel/domestic/service.json?from_postcode=2000&height=2.0&length=19.0&to_postcode=3108&weight=0.25&width=14.0'
```
Comment (jonathankwok) [inline test/unit/carriers/australia_post_test.rb]: Interesting. I don't expect the added `.0` to break things but you never really know.

Comment (benwah) [inline test/unit/carriers/australia_post_test.rb]: There are other test requests that include decimal values as part of the url params, so I assumed it was fine. We'll need to run remote tests to be certain.

File: lib/active_shipping/package.rb [hunk 2]
```diff
@@ -33,10 +31,10 @@ def initialize(grams_or_ounces, dimensions, options = {})
       @weight_unit_system = weight_imperial ? :imperial : :metric
       @dimensions_unit_system = dimensions_imperial ? :imperial : :metric
 
-      @weight = attribute_from_metric_or_imperial(grams_or_ounces, Mass, @weight_unit_system, :grams, :ounces)
+      @weight = attribute_from_metric_or_imperial(grams_or_ounces, Measured::Weight, @weight_unit_system, :grams, :ounces)
 
       if @dimensions.blank?
-        @dimensions = [Length.new(0, (dimensions_imperial ? :inches : :centimetres))] * 3
+        @dimensions = [Measured::Length.new(0, (dimensions_imperial ? :inches : :centimetres))] * 3
```
Comment (nicolaslupien) [inline lib/active_shipping/package.rb]: nit: this line is really hard to read

**Comment by kmcphillips** [comment]
Closes #427

File: lib/active_shipping/package.rb (hunk 2 above)
Comment (benwah) [inline lib/active_shipping/package.rb]: Made this a little more readable.

File: test/remote/ups_test.rb [hunk 3]
```diff
@@ -500,4 +502,23 @@ def test_create_shipment_with_insured_value
     assert response.success?
     assert_instance_of ActiveShipping::LabelResponse, response
   end
+
+  private
+
+  def with_ups_holidays
```
Comment (kmcphillips) [inline test/remote/ups_test.rb]: This is the right idea! Should be extracted into something more generic and part of the test env setup. Used with other carriers. But good to get this PR green. Maybe cut an issue to make sure we don't forget to generalize this.

Comment (jonathankwok) [inline test/remote/ups_test.rb]: Agree with everything here. This is pretty beautiful though, very readable.

**Comment by benwah** [comment]
repping @MalazAlamir @jonathankwok - Removed the `with_ups_holidays` with this https://github.com/Shopify/active_shipping/pull/452

**Comment by jonathankwok** [comment]
CI looks like a flaky remote failure, I retried it for you.

**Comment by benwah** [comment]
Damn 
image
tests

**Pull request merged**